site/content/en/docs/contrib/guide.en.md

```diff
@@ -1,52 +1,53 @@
 ---
 title: "Contributor Guide"
 linkTitle: "Guide"
-date: 2019-07-31
+date: 2019-11-09
```

Comment:
i m not sure what is this date is used for in the template,
but if we are updating it, it should be "2021"

Reply:
I have updated the date with 2021, and committed the change.